test/hotspot/gtest/metaspace/test_virtualspacenode.cpp

```diff
@@ -538,6 +539,8 @@ TEST_VM(metaspace, virtual_space_node_test_basics) {
   ASSERT_EQ(node->committed_words(), (size_t)0);
   ASSERT_EQ(node->committed_words(), scomm.get());
   DEBUG_ONLY(node->verify_locked();)
+
+  delete node;
```

Comment:
Can you please add an ASSERT after the delete to test that sres and scomm are 0?

Reply:
Done.